### DISCUSSION TITLE
Blog title points to the post, not the root

### DISCUSSION THREAD

**Discussion by talonx** [daattali/beautiful-jekyll#1231] [Q&A]
My website is at https://code.deepinspace.net/ and the code repo is https://github.com/talonx/talonx.github.io/

I switched to Beautiful Jekyll recently from Minima. The hyperlink for the blog title inside a post points to the post, not the root. 

Steps to view
- Go to https://code.deepinspace.net/
- Click on any post
- Hover the mouse over the blog title "On Code, and Other Things" - it points to the post whereas it should point to the blog's root.

I'm not able to figure out if this is a config error or something else. Any pointers?

**Comment by daattali** [comment]
When did you move to using this theme? Have you always observed this behaviour?

I ask because I just noticed last week a similar issue, and I committd a fix, I hope it works for you too (this was the fix https://github.com/daattali/beautiful-jekyll/commit/a206f70d118cd2873dea92480990ff7c62df5194)

In your config file, can you try to point at the latest (unreleased) version instead of 6.0.1?

Change this line:

```
remote_theme: daattali/beautiful-jekyll@6.0.1
```

to

```
remote_theme: daattali/beautiful-jekyll
```

Let me know

**Comment by talonx** [comment]
I moved to this theme a few days ago. 

The problem did not exist earlier. Also, it does not show up when I run it locally using `bundle exec jekyll serve`.

I'll try the latest version and let you know, thanks.

**Comment by daattali** [comment]
I think your problem is that your config file has `baseurl` and `url` parameters. Remove those.

**Reply by talonx** [reply]
That fixed it - thank you!
